db_test.go

```diff
@@ -748,7 +791,7 @@ func (db *TestDB) PrintStats() {
 
 // MustCheck runs a consistency check on the database and panics if any errors are found.
 func (db *TestDB) MustCheck() {
-	db.View(func(tx *bolt.Tx) error {
+	db.Update(func(tx *bolt.Tx) error {
```

Comment:
This was changed because calling `Tx.Check()` on a read-only transaction is not thread safe.